include/zephyr/posix/time.h

```diff
@@ -60,13 +60,14 @@ struct itimerspec {
 #include <errno.h>
```

Comment:
Commit message refers to k_clock instead of sys_clock

Reply:
fixed!